lib/perl/Genome/Model/Tools/Varscan/SomaticFilterWorkflow.pm

```diff
@@ -0,0 +1,231 @@
+package Genome::Model::Tools::Varscan::SomaticFilterWorkflow;
+
+use strict;
+use warnings;
+
+use Genome;
+use Workflow::Simple;
+use File::Basename;
+
+BEGIN {
+    $ENV{WF_USE_FLOW} = 1;
```

Comment:
I would set this at the top of `execute` with `local $ENV{WF_USE_FLOW} = 1;`.  That way it is only changed when the command is run.